exonum-java-binding-common/src/main/java/com/exonum/binding/common/proofs/map/UncheckedFlatMapProof.java

```diff
@@ -44,15 +48,18 @@
 
   private final List<MapEntry> entries;
 
-  private final List<byte[]> missingKeys;
+  private final List<ByteString> missingKeys;
 
   UncheckedFlatMapProof(
       List<MapProofEntry> proof,
       List<MapEntry> entries,
       List<byte[]> missingKeys) {
     this.proof = proof;
     this.entries = entries;
-    this.missingKeys = missingKeys;
+    // TODO: create ByteString in native
```

Comment:
No, native interface is supposed to be convenient for native code and efficient.

Reply:
Ok, I wasn't sure whether we should use ByteString or byte[] from native, thanks!